prime-router/src/test/kotlin/fhirengine/azure/FHIRConverterIntegrationTests.kt

```diff
@@ -108,7 +108,7 @@ class FHIRConverterIntegrationTests {
         return FHIRConverter(
             metadata,
             settings,
-            ReportStreamTestDatabaseContainer.testDatabaseAccess,
+            ReportStreamTestDatabaseContainer.testDatabaseAccess
```

Comment:
Are we not doing the trailing comma convention?

Reply:
I'm doing whatever ktlint does

Reply:
I don't really care that much, I just swear it yelled at me for _not_ doing trailing commas before 😂 and was trying to figure out which way we were supposed to be doing it